src/App.js

```diff
@@ -9,13 +9,26 @@ class App extends React.Component {
   constructor(){
     super()
     this.state={
-      data: movieData,
+      data: movieData.movies,
       currentView: "main",
       error: "",
       selectedMovie: 0
     };
   };
 
+  componentDidMount = () => {
+    return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+        .then ( response => response.json())
+        .then (movies => this.updateMovieData(movies.movies))
+        .catch(error => alert('Something went wrong.  Please try again later'));
+  }
+
+  updateMovieData = (movies) => {
+    this.setState({
+      data: movies
+    })
+  }
```

Comment:
Not sure if this is being used elsewhere yet, but if the only purpose of this method is to call it on `componentDidMount` in App, you can probably remove this method and call the `setState` on line 22.  I see it being passed as a prop to the `Display` component later, but it doesn't seem to be used yet (at least in this PR, so apologies if that has changed).